e2e/csi/ebs.go

```diff
@@ -75,6 +75,14 @@ func (tc *CSIControllerPluginEBSTest) BeforeAll(f *framework.F) {
 
 // AfterAll cleans up the volumes and plugin jobs created by the test.
 func (tc *CSIControllerPluginEBSTest) AfterAll(f *framework.F) {
+
+	// Stop all jobs in test
+	for _, id := range tc.testJobIDs {
+		out, err := e2e.Command("nomad", "job", "stop", "-purge", id)
+		f.Assert().NoError(err, out)
```

Comment:
Just random thought, we're all over the place with how we get at the `T` across e2e tests. Might be nice one day to normalize on one pattern.

Reply:
Totally agreed... having `require` on the `framework.F` is handy but it makes for inconsistency.